### ISSUE TITLE
Errors with Elixir 1.3.0

### ISSUE THREAD

**Issue by kmarsh** [Zensavona/elixtagram#8]
I just pulled and tried using this library. I got some dependency errors which I believe I was able to fix with:

``` diff
diff --git a/mix.exs b/mix.exs
index 2573692..133e71f 100644
--- a/mix.exs
+++ b/mix.exs
@@ -25,13 +25,13 @@ defmodule Elixtagram.Mixfile do

   defp deps do
     [
-      {:httpoison, "~> 0.7.4"},
-      {:poison, "~> 1.5"},
-      {:oauth2, "~> 0.5"},
-      {:exvcr, "~> 0.3", only: [:dev, :test]},
-      {:ex_doc, "~> 0.10.0", only: [:dev, :docs]},
+      {:httpoison, "~> 0.8"},
+      {:poison, "~> 2.2.0"},
+      {:oauth2, "~> 0.6"},
+      {:exvcr, "~> 0.7", only: [:dev, :test]},
+      {:ex_doc, "~> 0.12.0", only: [:dev, :docs]},
       {:excoveralls, "~> 0.3", only: [:dev, :test]},
-      {:inch_ex, "~> 0.4.0", only: [:dev, :docs]},
+      {:inch_ex, "~> 0.5.3", only: [:dev, :docs]},
       {:credo, "~> 0.1.0", only: :dev}
     ]
   end
```

However, tests fail with what looks like a deprecated syntax use in `elixtagram.ex:39`:

```
== Compilation error on file lib/elixtagram.ex ==
** (ArgumentError) defdelegate/2 only accepts function parameters, got: :global
    (elixir) lib/kernel/utils.ex:55: Kernel.Utils.add_arg/2
    (elixir) lib/kernel/utils.ex:25: Kernel.Utils.defdelegate/2
    lib/elixtagram.ex:39: anonymous fn/3 in :elixir_compiler_1.__MODULE__/1
    (elixir) lib/enum.ex:1622: Enum."-reduce/3-lists^foldl/2-0-"/3
    lib/elixtagram.ex:39: (module)
```

I wonder if it's related to https://github.com/elixir-lang/elixir/issues/4759 ?

Sorry I don't have more details or the ability to help fix this, I'm rather new to Elixir.

Thanks!

**Comment by Zensavona** [comment]
Hey @willcodeforfoo! Thanks for the bug report, I haven't actually upgraded to Elixir 1.3 but I'll do it today and investigate this :)

**Comment by deadkarma** [comment]
If you change the `:global` atom to a variable on this line https://github.com/Zensavona/elixtagram/blob/master/lib/elixtagram.ex#L39 it should work out

**Comment by Zensavona** [comment]
This is resolved in `0.2.8`
